src/routes/auth/login.route.js

```diff
@@ -0,0 +1,50 @@
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcryptjs')
+const crypto = require('crypto')
+const User = require('../../entities/user')
+
+module.exports.post = {
+	method: 'POST',
+	path: '/login',
+	schema: {
+		body: {
+			type: 'object',
+			additionalProperties: false,
+			required: ['username', 'password'],
+			properties: {
+				username: {
+					type: 'string',
+				},
+				password: {
+					type: 'string',
+				},
+			},
+		},
+	},
+	handler: async (req, res, _next) => {
+		const { username, password } = req.body
+
+		const user = await User.findOne({ username })
+		if (!user) {
+			return res.status(400).json({ message: 'Invalid username or password' })
+		}
+
+		const passwordMatch = await bcrypt.compare(password, user.password)
+		if (!passwordMatch) {
+			return res.status(400).json({ message: 'Invalid username or password' })
```

Comment:
i will implement a more specific error like `StatusErrorCustomMsg` for these errors to clean up